app/services/code_climate/link.rb

```diff
@@ -0,0 +1,52 @@
+module CodeClimate
+  class Link < BaseService
+    CODE_CLIMATE_API_ORG_NAME = ENV['CODE_CLIMATE_API_ORG_NAME']
+
+    def initialize(project)
+      @project = project
+    end
+
+    def call
+      return if code_climate_project_metric.blank? || !ids_differ
+
+      code_climate_project_metric.update!(cc_repository_id: external_cc_repository_id)
+
+      Rails.logger.info { message }
+    end
+
+    private
+
+    attr_reader :project
+
+    def code_climate_project_metric
+      @code_climate_project_metric ||= CodeClimateProjectMetric.find_by(project_id: project.id)
+    end
+
+    def ids_differ
```

Comment:
when methods return boolean values we usually use names ending in `?`, in this case `ids_differ?`